requirements.txt

```diff
@@ -1,2 +1,4 @@
 jsonschema
+pygal
+jinja2
```

Comment:
Thanks for this in particular -- I'll need to update our requirements list with our licensing folk and it's helpful to have these earlier rather than later.